tests/acceptance/expected-failures-on-OCIS-storage.md

```diff
@@ -1077,87 +1062,87 @@ Scenario Outline: A disabled user cannot use webdav
 -   [apiAuthWebDav/webDavDELETEAuth.feature:162](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiAuthWebDav/webDavDELETEAuth.feature#L162)
 
 #### [various sharing settings cannot be set](https://github.com/owncloud/ocis/issues/1328)
--   [apiCapabilities/capabilities.feature:8](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiCapabilities/capabilities.feature#L8)
```

Comment:
thee are lots of lines here with only some spaces removed. Those need to be reverted @kiranparajuli589

Reply:
this silly IDE keeps removing the spaces wherever I dropped the cursor.

reverted @phil-davis, please review again!